frameworks/AutoGluon/exec.py

```diff
@@ -68,6 +68,17 @@ def run(dataset, config):
     label = dataset.target.name
     problem_type = dataset.problem_type
 
+    # whether to generate learning curves (EXTREMELY EXPENSIVE)
+    if "learning_curves" in training_params:
+        _curve_metrics = config.framework_params.get('_curve_metrics', None)
+        if "metrics" not in training_params["learning_curves"] and problem_type in _curve_metrics:
```

Comment:
Will crash if `_curve_metrics=None` in the `if` check, since `in None` will raise an exception.

Instead:
_curve_metrics = config.framework_params.get('_curve_metrics', {})